synctl/cli.py

```diff
@@ -331,6 +332,119 @@ def exit_synctl(self, error_code=-1, message=''):
             print(message)
         sys.exit(error_code)
 
+class PopConfiguration(Base):
+    def __init__(self) -> None:
+        Base.__init__(self)
+        self.agent = {
+            "cpuLimit" : 1500,
+            "memLimit" : 768,
+            "imageSize" : 600,
+        }
+        self.k8ssensor = {
+            "cpuLimit" : 500,
+            "memLimit" : 1536,
+            "imageSize" : 80,
+        }
+        self.controller = {
+            "cpuLimit" : 300,
+            "memLimit" : 300,
+            "imageSize" : 900,
+        }
+        self.redis = {
+            "cpuLimit" : 300,
+            "memLimit" : 200,
+            "imageSize" : 500,
+        }
+        self.http = {
+            "testCount" : 2000,
+            "frequency": 1,
+            "cpuLimit" : 300,
+            "memLimit" : 500,
+            "imageSize" : 400,
+        }
+        self.javascript = {
+            "testCount" : 20,
+            "frequency": 1,
+            "cpuLimit" : 800,
+            "memLimit" : 300,
+            "imageSize" : 400,
+        }
+        self.browserscript = {
+            "testCount" : 5,
+            "frequency": 5,
+            "cpuLimit" : 4000,
+            "memLimit" : 3000,
+            "imageSize" : 1500,
+        }
+
+    def ask_question(self,question, options=None):
+        answer = input(question)
+        if options:
+            while answer not in options:
+                print("Invalid input.")
+                answer = input(question)
+        return answer
+
+    def size_estimate(self, user_tests, default_frequency, user_frequency, default_tests):
+
+        pod_estimate = int(user_tests * default_frequency) / int(user_frequency * default_tests)
+        return math.ceil(pod_estimate)
+
+
+    def pop_size_estimate(self):
+        print("Please answer below questions for estimating the self-hosted PoP hardware size\n")
+        try:
+            api_simple = int(self.ask_question("How many API Simple tests do you want to create? (0 if no) "))
+            if api_simple != 0:
+                api_simple_frequency = int(self.ask_question("What is the test frequency for your API Simple tests ? (1-120)  "))
```

Comment:
check the value of `api_simple_frequency` here, it should be between 1 and 120, any value other than [1, 120] should output a warning message and ask user to input it again.